test_suite/package/test_awscli2.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+import pytest
+
+from lib import test_lib
+
+
+@pytest.mark.package
+@pytest.mark.run_on(['>=rhel9.5'])
+class TestsAwsCli2:
+    @pytest.fixture(scope='module', autouse=True)
+    def import_aws_credentials(self, host):
+        # Generate temporary credentials for this test.
+        civ_local_command_to_run = 'aws sts get-session-token --duration-seconds 3600 --output json'
```

Comment:
Tests take less than 1 minute to run, so maybe we should reduce this number to something much smaller? E.g. `90`

Reply:
I think reducing the number to something much smaller is a good idea too. I'd probably set it to 90 like you suggested or maybe 120 to be extra cautious!

Reply:
```suggestion
        civ_local_command_to_run = 'aws sts get-session-token --duration-seconds 120 --output json'
```